### DISCUSSION TITLE
helm chart is broken

### DISCUSSION THREAD

**Discussion by didlawowo** [open-webui/open-webui#1156] [General]
# Bug Report

## Description

generate chart from kompose replace good value with some errored image

**Bug Summary:**
image pull back off on webui and storage error on ollama

**Steps to Reproduce:**
helm install 

**Expected Behavior:**
working deployed pod

**Actual Behavior:**

## Environment

- **Operating System:** [K3s]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

 
time=2024-03-13T13:36:08.352Z level=INFO source=images.go:800 msg="total blobs: 0"                                                                              │
│ Error: mkdir /root/.ollama/models/manifests: no space left on device                                                                                            │
│ Stream closed EOF for mlops/ollama-79c7468c97-xz2dr (ollama)    

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Moving the discussion as helm is community supported method of installation, but if I had to guess `no space left on device` seems to be the culprit here.
